<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_character</class>
 <widget class="QDialog" name="change_character">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change character</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Character:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="char_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New character:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="character_val">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>175</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>change_character</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
